### ISSUE TITLE
Sub-dependencies need to be built

### ISSUE THREAD

**Issue by rchl** [DataDog/dd-trace-js#1554]
**Describe the bug**

This is not so much a bug but rather a complain and hope for some workaround.

After updating from version `1.1.0` to `1.1.1`, a bunch of new dependencies from the `@stdlib` namespace are added. Some of those need to be compiled while installing dependencies and since our minimal docker image doesn't contain `make` and related libraries, it fails with:

```
[2021-08-05T13:11:37.420Z] > dd-trace@1.1.1 preinstall /app/node_modules/dd-trace
[2021-08-05T13:11:37.420Z] > node scripts/preinstall.js
[2021-08-05T13:11:37.420Z]
[2021-08-05T13:11:37.420Z]
[2021-08-05T13:11:37.420Z] > @stdlib/math-base-assert-is-infinite@0.0.7 install /app/node_modules/@stdlib/math-base-assert-is-infinite
[2021-08-05T13:11:37.420Z] > node-gyp rebuild
[2021-08-05T13:11:37.420Z]
[2021-08-05T13:11:37.420Z] gyp ERR! build error
[2021-08-05T13:11:37.420Z] gyp ERR! stack Error: not found: make
[2021-08-05T13:11:37.420Z] gyp ERR! stack     at getNotFoundError (/usr/local/lib/node_modules/npm/node_modules/which/which.js:13:12)
[2021-08-05T13:11:37.420Z] gyp ERR! stack     at F (/usr/local/lib/node_modules/npm/node_modules/which/which.js:68:19)
[2021-08-05T13:11:37.420Z] gyp ERR! stack     at E (/usr/local/lib/node_modules/npm/node_modules/which/which.js:80:29)
[2021-08-05T13:11:37.420Z] gyp ERR! stack     at /usr/local/lib/node_modules/npm/node_modules/which/which.js:89:16
[2021-08-05T13:11:37.420Z] gyp ERR! stack     at /usr/local/lib/node_modules/npm/node_modules/isexe/index.js:42:5
[2021-08-05T13:11:37.420Z] gyp ERR! stack     at /usr/local/lib/node_modules/npm/node_modules/isexe/mode.js:8:5
[2021-08-05T13:11:37.420Z] gyp ERR! stack     at FSReqCallback.oncomplete (fs.js:192:21)
[2021-08-05T13:11:37.420Z] gyp ERR! System Linux 5.8.0-1041-aws
[2021-08-05T13:11:37.420Z] gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
[2021-08-05T13:11:37.420Z] gyp ERR! cwd /app/node_modules/@stdlib/math-base-assert-is-infinite
[2021-08-05T13:11:37.420Z] gyp ERR! node -v v14.17.4
[2021-08-05T13:11:37.420Z] gyp ERR! node-gyp -v v5.1.0
[2021-08-05T13:11:37.420Z] gyp ERR! not ok
[2021-08-05T13:11:37.420Z] npm ERR! code ELIFECYCLE
[2021-08-05T13:11:37.420Z] npm ERR! errno 1
[2021-08-05T13:11:37.420Z] npm ERR! @stdlib/math-base-assert-is-infinite@0.0.7 install: `node-gyp rebuild`
[2021-08-05T13:11:37.420Z] npm ERR! Exit status 1
[2021-08-05T13:11:37.420Z] npm ERR!
[2021-08-05T13:11:37.420Z] npm ERR! Failed at the @stdlib/math-base-assert-is-infinite@0.0.7 install script.
[2021-08-05T13:11:37.420Z] npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
[2021-08-05T13:11:37.420Z]
```

Is there anything that could be done here?
I guess that package could provide pre-built libraries but this is probably not in control of datadog.
Alternatively, maybe the datadog components could not use those dependencies.

**Environment**

* **Operation system:** 
* **Node version:** v14
* **Tracer version:** v1.1.1
* **Agent version:**

**Dockerfile:**

```
FROM node:14-slim

# Install curl required by side-containers
RUN apt-get update && \
    apt-get install -y --no-install-recommends ca-certificates curl netbase python3 wget && \
    rm -rf /var/lib/apt/lists/*
```

**Comment by rochdev** [comment]
We are in the process of removing this dependency directly in `sketches-js`.

**Comment by rochdev** [comment]
Just released `sketches-js` 1.0.4 which should solve the issue. Simply reinstalling should work since it will pull the new version as it matches the dependency version range. I will also bump the dependency.
